### ISSUE TITLE
Onewheel+ unable to view cell voltages

### ISSUE THREAD

**Issue by tiztrain** [OnewheelCommunityEdition/OWCE_App#60]
Have a onewheel+ with CB+ battery and I am unable to view the voltages of the different cells.

Thought I should log this. Not sure if it is supposed to work?

**Comment by beeradmoore** [comment]
From what I've been told, CB+ has it's own BMS so it doesn't send information through to the controller and then through to the app. Do you get to see the entire pack voltage at least?

In future if there is ever a menu to select your battery type (for correct battery % to be displayed) it would be nice if selecting CB+ would tell you this.

**Comment by tiztrain** [comment]
Yes you are correct. A BMS is supplied in it. Can see the entire voltage :)

That would be awesome if you could do that. Having charging battery issues and thought this might show if everything is balanced
